### PR TITLE
adding SWEpy

### PR DESCRIPTION
**Pull request by wino6687**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [ ] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by wino6687** [comment]
I am the maintainer, and I am willing to maintain SWEpy in the future

File: recipes/swepy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+requirements:
+  build:
+
+  host:
+    - python
+    - pip
+  run:
+    - matplotlib
+    - python=3.6
```
Comment (scopatz) [inline recipes/swepy/meta.yaml]: Is this really python 3.6 only?  Also there needs to be space here `python =3.6`

File: recipes/swepy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    - pytest
+    - pandas
+    - requests
+    - tqdm
+    - pynco
+    - gdal
+    - scikit-image
+    - affine
+    - netCDF4
```
Comment (scopatz) [inline recipes/swepy/meta.yaml]: package names are all lowercase

File: recipes/swepy/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 91ed36a907399ce7a29d763cf74077ce14d55e8bcdf52b6f1e8b5723f48d322f
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
```
Comment (scopatz) [inline recipes/swepy/meta.yaml]: This should be `{{ PYTHON }} -m pip install --no-deps --ignore-installed .`

File: recipes/swepy/meta.yaml (hunk 1 above)
Comment (wino6687) [reply recipes/swepy/meta.yaml]: @scopatz, it also runs in python 3.7. But I was unsure of how to add both to the meta.yaml file? The NetCDF Operators (NCOpy) are only testing in 3.6/3.7

File: recipes/swepy/meta.yaml (hunk 2 above)
Comment (wino6687) [reply recipes/swepy/meta.yaml]: fixed, thank you!

File: recipes/swepy/meta.yaml (hunk 1 above)
Comment (scopatz) [reply recipes/swepy/meta.yaml]: All you need to do here is `python >=3.6`

File: recipes/swepy/meta.yaml (hunk 3 above)
Comment (wino6687) [reply recipes/swepy/meta.yaml]: This has been fixed as well! Thanks for the assistance

File: recipes/swepy/meta.yaml (hunk 1 above)
Comment (wino6687) [reply recipes/swepy/meta.yaml]: Sorry about that, committed it yesterday but forgot to push it.

**Comment by wino6687** [comment]
@scopatz, thanks for your time. The python version should be fixed now

**Pull request merged**